src/modules/utils.js

```diff
@@ -82,6 +82,21 @@ export const inputBottomPadding = () => {
   }
 }
 
+// will take the metaTokens property on the wallet (that comes from currencyInfo), merge with account-level custom tokens added, and only return if enabled (wallet-specific)
```

Comment:
there are libraries capable of doing this for you
we already use Lodash, so feel free to use anything that library provides
[_.intersectionWith](https://lodash.com/docs/4.17.4#intersectionWith) would likely do the trick
```
var objects = [{ 'x': 1, 'y': 2 }, { 'x': 2, 'y': 1 }];
var others = [{ 'x': 1, 'y': 1 }, { 'x': 1, 'y': 2 }];
 
_.intersectionWith(objects, others, _.isEqual);
// => [{ 'x': 1, 'y': 2 }]
```

Reply:
I will familiarize myself with that library. I was not aware that we were already using it.